docs/tutorials/get-started/try-viam-sdk.md

```diff
@@ -540,8 +540,8 @@ package main
 import (
     "context"
 
-    "github.com/edaniels/golog"
     "go.viam.com/rdk/components/base"
+    "go.viam.com/rdk/logging"
```

Comment:
I see 4 usages of golog in this file to change to logging. Lines:
* 296
* 324
* 550
* 562